tenable/ad/users/api.py

```diff
@@ -0,0 +1,285 @@
+'''
+Users
+=============
+
+Methods described in this section relate to the users API.
+These methods can be accessed at ``TenableAD.users``.
+
+.. rst-class:: hide-signature
+.. autoclass:: UsersAPI
+
+    .. automethod:: list
+    .. automethod:: create
+    .. automethod:: info
+    .. automethod:: details
+    .. automethod:: update
+    .. automethod:: delete
+    .. automethod:: create_password
+    .. automethod:: retrieve_password
+    .. automethod:: change_password
+    .. automethod:: update_user_roles
```

Comment:
this could be replaced with `:members:` instead.

Example:

https://github.com/tenable/pyTenable/blob/c6d2f67cd55821ed940399c3a401c1960e47c062/tenable/io/agent_config.py#L10-L12

Reply:
replaced `automethod` with `:members:`